ftw/noticeboard/browser/templates/notice.pt

```diff
@@ -4,27 +4,42 @@
       xmlns:i18n="http://xml.zope.org/namespaces/i18n"
       i18n:domain="ftw.noticeboard">
 
+    <head>
+        <div metal:fill-slot="javascript_head_slot" tal:omit-tag="">
+            <script type="text/javascript">
+              (function($){
+                const config = {
+                    adaptiveHeight: true,
+                }
+                $(document).ready(function(){
+                  $('.notice-image-slider').slick(config);
+                });
+              }(jQuery));
+            </script>
+        </div>
+    </head>
+
     <div metal:fill-slot="main">
         <div class="notice-detail-wrapper">
```

Comment:
Thanks for putting a wrapper around everything! It will help a lot with styling the view.